app-arch/innoextract/innoextract-1.8.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
7 should work here.

Reply:
I have updated the PR with the suggested changes.